### PR TITLE
[WIP] Stun equipment rework.

### PR DESCRIPTION
**Pull request by TheChosenEvilOne**
Changes to how certain stunning tools work (such as the flash, taser, and stun baton), https://hackmd.io/THAd3pX-RPCkp6Y3y-yjjQ
- Flash
  - No longer stuns on flash instead when a flashed person walks in to something they get stunned.
- Taser
  - Instead of stunning causes 50% slowdown and increases stamina damage taken.
- Stun baton
  - Delayed knockdown removed.
- Pepperspray
  - Spray speed increased, should be harder to dodge now.

### CODE REVIEW

**Comment by JinGiant** [comment]
Dear TCEO,

Hello, please update the description of your pr to differentiate between the telescopic baton and stun baton

Kind regards,
Jin

**Comment by TheChosenEvilOne** [comment]
> Dear TCEO,
> 
> Hello, please update the description of your pr to differentiate between the telescopic baton and stun baton
> 
> Kind regards, Jin

fine..

**Comment by JinGiant** [comment]
> fine..

Dear TCEO,

Thank you for updating the description, I have changed my emoticon reaction to reflect that

Kind regards,
Jin

**Comment by EdgeLordExe** [comment]
@TheChosenEvilOne the server exploded in popularity wake up

File: code/modules/assembly/flash.dm [hunk 1]
```diff
@@ -192,61 +179,13 @@
...
-	// Attacker at 135 or more degrees of where the victim is facing.
-	if(victim_dir & REVERSE_DIR(victim_to_attacker))
-		return DEVIATION_FULL
-
-	// - - -
-	// # V # Victim facing south
-	// - - -
-	// Attacker lateral to the victim.
-	return DEVIATION_PARTIAL
-
```
Comment (EdgeLordExe) [inline code/modules/assembly/flash.dm:242]: i liked this mechanic, think you can bring it back in some way?

**Comment by EdgeLordExe** [comment]
besides that one comment LGTM

**Comment by karmaisblackandbluepilled** [comment]
Tbh the flash changes suck ass. The flash acts like a ranged stun now, since you are p.much 100% guaranteed to walk over an item and into a wall and get fully stunned. Go back to flashes being stuns and only blinding and disorienting you, without the stun and tripping.

**Comment by karmaisblackandbluepilled** [comment]
Also, the taser is just straight up worse than a disabler at this point. Bring back hippie tasers that stunned you in one hit.

**Comment by EtraIV** [comment]
You should probably write a changelog for this now that we have automatic CL generation. Also, given how this is a balance change, and is actually being test merged, you should follow the template and actually justify _why_ it is good for the game.

**Comment by karmaisblackandbluepilled** [comment]
Tazer still sounds like ass, the point is to not have to get closer so you don't get shove shitted. 
Make the flash work like before again, just a plain ass stun.

**Comment by QuakeIV** [comment]
im inclined to close this, change sounds gay and TCEO fucked off, any objections? if not ill yeet it in a week or something

**Comment by QuakeIV** [comment]
yeet

**Comment by EtraIV** [comment]
reopening just so tgs doesn't shit itself, will reclose

**Pull request closed without merging**